ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -112,7 +117,8 @@ export default function EditGasPopover({
   );
 
   const txParamsHaveBeenCustomized =
-    estimateToUse === 'custom' || txParamsAreDappSuggested(updatedTransaction);
+    estimateToUse === CUSTOM_GAS_ESTIMATE ||
```

Comment:
```suggestion
    (estimateToUse === CUSTOM_GAS_ESTIMATE) ||
```
more explicit order of operations

Reply:
`yarn lint:fix` removes those parens, so I think we're good.